src/browser/modules/Editor/Monaco.tsx

```diff
@@ -446,33 +447,32 @@ const Monaco = forwardRef<MonacoHandles, MonacoProps>(
         bus.self(
           CYPHER_REQUEST,
           {
-            query: 'EXPLAIN ' + text,
+            query: EXPLAIN_QUERY_PREFIX + text,
             queryType: NEO4J_BROWSER_USER_ACTION_QUERY
           },
-          response => {
+          (response: { result: QueryResult; success?: boolean }) => {
             if (
               response.success === true &&
               response.result.summary.notifications.length > 0
             ) {
               editor.setModelMarkers(model, monacoId, [
                 ...editor.getModelMarkers({ owner: monacoId }),
                 ...response.result.summary.notifications.map(
-                  ({
-                    description,
-                    position: { line },
-                    title
-                  }: {
-                    description: string
-                    position: { line: number }
-                    title: string
-                  }) => ({
-                    startLineNumber: statementLineNumber + line,
-                    startColumn: 1,
-                    endLineNumber: statementLineNumber + line,
-                    endColumn: 1000,
-                    message: title + '\n\n' + description,
-                    severity: MarkerSeverity.Warning
-                  })
+                  ({ description, position, title }) => {
+                    const { line, column } = position as NotificationPosition
```

Comment:
Seems from the types that position could be `{}` and that's why we cast to NotificationPosition is that not a case we need to handle?

Reply:
I have not seen it happen, the type info might be incorrect, but I've refactored this line so accessing line and column is type safe now.